orcid-core/src/main/resources/i18n/email_ja.properties

```diff
@@ -368,53 +368,55 @@ email.claim_reminder.within_the_next.3=\u0020 があなたの代わりに入力
                                        ブラウザーに貼り付けます。\n\
 
 # Service announcement_1_2015
-email.service_announcement.body_intro=Today, we are writing to let you know about changes to how we contact you, how we will notify you if specific organizations seek your permission to help maintain your ORCID record, and amendments to our privacy policy that will help us identify and block spammers. What has not changed is our\u0020
-email.service_announcement.privacy_link=commitment to your privacy
-email.service_announcement.dot_bottom=\u002E
-email.service_announcement.body_inbox_title=New:  The ORCID Inbox
-email.service_announcement.body_inbox1=To help you manage how and when you receive notifications from ORCID, we have developed an ORCID Inbox system,\u0020
+email.service_announcement.dear1=
+email.service_announcement.dear2=\u0020\u69D8
+email.service_announcement.body_intro=\u3054\u767B\u9332\u8005\u69D8\u3078\u306E\u3054\u9023\u7D61\u65B9\u6CD5\u306E\u5909\u66F4\u3001ORCID\u30EC\u30B3\u30FC\u30C9\u306E\u4FDD\u5168\u306E\u305F\u3081\u306B\u7279\u5B9A\u306E\u7D44\u7E54\u304C\u8A31\u53EF\u3092\u6C42\u3081\u305F\u5834\u5408\u306E\u3054\u901A\u77E5\u65B9\u6CD5\u3001\u30B9\u30D1\u30E0\u30E1\u30FC\u30EB\u9001\u4FE1\u8005\u306E\u7279\u5B9A\u3068\u30D6\u30ED\u30C3\u30AF\u306B\u95A2\u3059\u308B\u5F53\u793E\u306E\u500B\u4EBA\u60C5\u5831\u4FDD\u8B77\u65B9\u91DD\u306E\u6539\u6B63\u306B\u3064\u3044\u3066\u304A\u77E5\u3089\u305B\u3044\u305F\u3057\u307E\u3059\u3002 \u3054\u767B\u9332\u8005\u69D8\u306E
```

Comment:
Um I don't think you know what I mean by UTF8? Not \uXXXX, that's to hard
to read.

On Sat, Oct 10, 2015 at 9:02 PM, Shobhit Tyagi <notifications@github.com>
wrote:

> In orcid-core/src/main/resources/i18n/email_ja.properties
> <https://github.com/ORCID/ORCID-Source/pull/2347#discussion_r41704952>:
>
> > @@ -368,53 +368,55 @@ email.claim_reminder.within_the_next.3=\u0020 があなたの代わりに入力
> >                                         ブラウザーに貼り付けます。\n\
> >
> >  # Service announcement_1_2015
> > -email.service_announcement.body_intro=Today, we are writing to let you know about changes to how we contact you, how we will notify you if specific organizations seek your permission to help maintain your ORCID record, and amendments to our privacy policy that will help us identify and block spammers. What has not changed is our\u0020
> > -email.service_announcement.privacy_link=commitment to your privacy
> > -email.service_announcement.dot_bottom=\u002E
> > -email.service_announcement.body_inbox_title=New:  The ORCID Inbox
> > -email.service_announcement.body_inbox1=To help you manage how and when you receive notifications from ORCID, we have developed an ORCID Inbox system,\u0020
> > +email.service_announcement.dear1=
> > +email.service_announcement.dear2=\u0020\u69D8
> > +email.service_announcement.body_intro=\u3054\u767B\u9332\u8005\u69D8\u3078\u306E\u3054\u9023\u7D61\u65B9\u6CD5\u306E\u5909\u66F4\u3001ORCID\u30EC\u30B3\u30FC\u30C9\u306E\u4FDD\u5168\u306E\u305F\u3081\u306B\u7279\u5B9A\u306E\u7D44\u7E54\u304C\u8A31\u53EF\u3092\u6C42\u3081\u305F\u5834\u5408\u306E\u3054\u901A\u77E5\u65B9\u6CD5\u3001\u30B9\u30D1\u30E0\u30E1\u30FC\u30EB\u9001\u4FE1\u8005\u306E\u7279\u5B9A\u3068\u30D6\u30ED\u30C3\u30AF\u306B\u95A2\u3059\u308B\u5F53\u793E\u306E\u500B\u4EBA\u60C5\u5831\u4FDD\u8B77\u65B9\u91DD\u306E\u6539\u6B63\u306B\u3064\u3044\u3066\u304A\u77E5\u3089\u305B\u3044\u305F\u3057\u307E\u3059\u3002 \u3054\u767B\u9332\u8005\u69D8\u306E
>
> @rcpeters <https://github.com/rcpeters> I will update the existing keys
> to unicode separately. For now just the service announcement ones are UTF-8
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ORCID/ORCID-Source/pull/2347/files#r41704952>.
>



-- 
Robert Peters
info@rcpeters.com
805.440.9056